### ISSUE TITLE
: mark causes problem on ArchLinux

### ISSUE THREAD

**Issue by kuppo** [ianka/xfs_undelete#7]
Hi,

Thanks for the brilliant tool.

I found an issue on my ArchLinux machine. dd complained the invalid argument cause the `of` parameter contains `:`, I believe the code is on line 83
```
83  set of [file join [dict get $::parameters o] [format "%s_%s" [clock format $ctime -format "%Y-%m-%d-%H:%M"] $inode]]
```
After I changed `"%Y-%m-%d-%H:%M"` to `"%Y-%m-%d-%H-%M"`, it worked normally.

I don't know if it is a bug, but good to have you known it.

**Comment by ianka** [comment]
Could you please do

    $ dd --version

and post the result?

**Comment by ianka** [comment]
fixed in v7.1
